conference.js

```diff
@@ -571,8 +600,18 @@ export default {
                             }
                         })
                         .then(onTracksCreated)
-                        .catch(() => {
-                            // can't do anything in this case, so just ignore;
+                        .catch((err) => {
+                            videoOnlyError = err;
+
+                            if (videoOnlyError) {
+                                APP.UI.showDeviceErrorDialog(
+                                    'camera', videoOnlyError);
+                            }
+
+                            if (audioOnlyError) {
+                                APP.UI.showDeviceErrorDialog(
+                                    'microphone', audioOnlyError);
```

Comment:
I'm surely missing something here. I have 2 alternative scenarios in which I have questions:

1. videoOnlyError means that there is video only (i.e. there's no audio) and there is an error. Consequently in this scenario, audioOnlyError means that there is audio only (i.e. there's no video) and there is an error. Then, one and only one of videoOnlyError and audioOnlyError could possibly be true (because if there's video only, then there's no audio, etc.). And then there's no point in doing "if (videoOnlyError); if (audioOnlyError);" but rather "if (videoOnlyError) else if (audioOnlyError)"?

2. videoOnlyError and audioOnlyError mean something else, different from scenario 1 and both of them can be true at the same time (because we test them with 2 separate ifs). But does that mean that we'll show 2 ErrorDialogs?